Collect 1.4.13 crash when selecting My Drive to fetch form

#### Software and hardware versions 
Collect v1.4.13, Android v5.1.1, device used Samsung Galaxy J1 Mini and J1

#### Problem description
ODK Collect crashes when we try to fetch an xml form from Google Drive.
#### Steps to reproduce the problem
1. Load Collect 1.4.13 from Google Play
2. Specify platform (Google Drive0
3. Specify Google account.
4. Get Blank Form (enters search GUI)
5. Select My Drive
6. Crash reported
7. Settings file attached
[collect.settings.txt](https://github.com/opendatakit/collect/files/729998/collect.settings.txt)

#### Expected behavior
Goto xml files in My Drive or Shared With Me and select relevant

#### Other information 
Tried on three makes of phone with 3 different versions
Samsung J1 4.44
Huawei P8 Lite 5.0
Samsung J1 Mini 5.1.1
Reverted to 1.4.12 APK and all works fine

The stack traces don't appear to have any useful info. Listing from the drive:
```
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$3.done(AsyncTask.java:309)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
	at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object com.google.a.b.a.a$b$b.e()' on a null object reference
	at org.odk.collect.android.activities.GoogleDriveActivity$b.a(Unknown Source)
	at org.odk.collect.android.activities.GoogleDriveActivity$b.doInBackground(Unknown Source)
	at android.os.AsyncTask$2.call(AsyncTask.java:295)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	... 3 more
```

Publishing a previously loaded form:
```
java.lang.RuntimeException: An error occurred while executing doInBackground()
	at android.os.AsyncTask$3.done(AsyncTask.java:309)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
	at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.NoSuchMethodError: No static method newPullParser()Lorg/g/a/a; in class Landroid/util/Xml; or its super classes (declaration of 'android.util.Xml' appears in /system/framework/framework.jar:classes2.dex)
	at org.odk.collect.android.h.g.a(Unknown Source)
	at org.odk.collect.android.h.g.a(Unknown Source)
	at org.odk.collect.android.h.g.a(Unknown Source)
	at org.odk.collect.android.activities.GoogleSheetsUploaderActivity$a.a(Unknown Source)
	at org.odk.collect.android.activities.GoogleSheetsUploaderActivity$a.doInBackground(Unknown Source)
	at android.os.AsyncTask$2.call(AsyncTask.java:295)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	... 3 more
```

@grzesiek2010 is probably correct that it's some kind of API auth issue like in #289 (the stack traces are also similar). @yanokwa and @mitchellsundt are working on it.

--- a/collect_app/build.gradle
+++ b/collect_app/build.gradle
@@ -39,15 +39,17 @@ android {
         applicationId("org.odk.collect.android")
         minSdkVersion(9)
         targetSdkVersion(9)
-        versionCode(1064)
+        versionCode(1065)
         versionName getVersionName()
         testInstrumentationRunner("android.support.test.runner.AndroidJUnitRunner")
     }
 
     buildTypes {
         release {
-            minifyEnabled(true)
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
+            // Proguard rules will need to be carefully revised before turning on minification --
+            // the current configuration breaks Google Drive API.
+            // minifyEnabled(true)
+            // proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
         }
         debug {
             debuggable(true)

